src/octoprint/util/comm.py

```diff
@@ -1752,7 +1752,7 @@ def _on_M400_sent():
             if self.isSdFileSelected():
                 if not external_sd:
                     self.sendCommand(
-                        "M25",
+                        settings().get(["serial", "sdCancelCommand"]),
```

Comment:
This should be read into a variable once at instantiation of the comm instance. See how `self._hello_command` is done.

Reply:
requested adjustments have been made